app/src/main/res/values/strings.xml

```diff
@@ -1707,6 +1707,12 @@
     <string name="source_wizard_button">Source Wizard Button</string>
     <string name="please_wait_seconds_before_trying_to_start_sensor">Please wait %d seconds before trying to start sensor</string>
     <string name="please_update_OOP2_or_move_to_calibrate_based_on_raw_mode">please update OOP2 or move to calibrate based on raw mode</string>
+    <string name="reminder">Reminder</string>
+    <string name="notification">Notification</string>
+    <string name="TX_EOL_reminder">When Transmitter Days on system status page reaches 99, it will be the last opportunity to start a sensor on this transmitter. Currently at %d days.</string>
+    <string name="TX_EOL_notification">If you proceed and start sensor it is very likely it will fail because of Transmitter Days having reached maximum.</string>
```

Comment:
If you proceed and start sensor, it is ...

I suggest adding a comma after sensor.